src/Administration/Resources/app/administration/src/module/sw-customer/snippet/de-DE.json

```diff
@@ -17,28 +17,28 @@
       "placeholderFirstName": "Vorname",
       "placeholderLastName": "Nachname",
       "placeholderCompany": "Firma",
-      "placeholderEmail": "E-Mail",
+      "placeholderEmail": "E-Mail-Adresse",
       "labelSalutation": "Anrede",
       "labelTitle": "Titel",
       "labelFirstName": "Vorname",
       "labelLastName": "Nachname",
       "labelCompany": "Firma",
-      "labelEmail": "E-Mail",
+      "labelEmail": "E-Mail-Adresse",
       "labelTags": "Tags"
     },
     "list": {
       "textCustomers": "Kunden",
       "textCustomersTotal": "Kein Kunde | Ein Kunde | {count} Kunden",
       "buttonAddCustomer": "Kunden anlegen",
-      "columnCustomerNumber": "Kunden-Nr.",
+      "columnCustomerNumber": "Kundennummer",
```

Comment:
did you check the module if the longer texts fits the column?

Reply:
Didn't see any need to check, while the english one is longer with `Customer number`